dpti/hti_liq.py

```diff
@@ -598,6 +598,17 @@ def add_module_subparsers(main_subparsers):
     )
     parser_compute.set_defaults(func=handle_compute)
 
+    parser_run = module_subparsers.add_parser("run", help="run the job")
+    parser_run.add_argument("JOB", type=str, help="folder of the job")
+    parser_run.add_argument("machine", type=str, help="machine.json file for the job")
+    parser_run.add_argument(
+        "task_name", type=str, help="task name, can be 00, 01, or 02"
+    )
+    parser_run.add_argument(
+        "--use-dp", type=bool, default=True, help="whether to use Deep Potential or not"
+    )
```

Comment:
**Suggestion:** The `--use-dp` argument in the `parser_run` command is a boolean. It's more common to use `store_true` or `store_false` for boolean command line arguments. This way, the presence of the argument sets the value, rather than requiring an explicit True or False. [best practice]
```suggestion
    parser_run.add_argument(
        "--use-dp", action="store_true", default=True, help="whether to use Deep Potential or not"
    )
```

Reply:
This is correct. However, if the default of `use_dp` is `True`, the argument should be `--do-not-use-dp` with `store_false`.